openapi.json

```diff
@@ -519,6 +599,181 @@
+                          "channel": {
+                            "type": "string",
+                            "enum": ["Safaricom", "Airtel"],
+                            "description": "Network channel associated with the balance"
+                          },
+                          "balance": {
+                            "type": "number",
+                            "format": "float",
+                            "description": "Current balance amount"
+                          },
+                          "allotted": {
+                            "type": "number",
+                            "format": "float",
+                            "description": "Total amount allotted to the balance"
+                          },
+                          "reserved": {
+                            "type": "number",
+                            "format": "float",
+                            "description": "Amount reserved from the balance"
+                          },
+                          "expiry": {
+                            "type": "string",
+                            "format": "date-time",
+                            "description": "Expiry date of the balance"
+                          }
+                        }
+                      }
+                    },
+                    "validity": {
+                      "type": "string",
+                      "description": "Validity period of the bundle",
+                      "nullable": true
+                    },
+                    "createdBy": {
+                      "type": "object",
+                      "properties": {
+                        "id": {
+                          "type": "number",
+                          "description": "Unique identifier for the user"
+                        },
+                        "email": {
+                          "type": "string",
+                          "description": "Email address of the user"
+                        },
+                        "firstName": {
+                          "type": "string",
+                          "description": "First name of the user"
+                        },
+                        "lastName": {
+                          "type": "string",
+                          "description": "Last name of the user",
+                          "nullable": true
+                        },
+                        "phone": {
+                          "type": "string",
+                          "description": "Phone number of the user",
+                          "nullable": true
+                        },
+                        "salespersonId": {
+                          "type": "string",
+                          "description": "Identifier for the salesperson",
+                          "nullable": true
+                        }
+                      },
+                      "description": "User who created the bundle"
+                    },
```

Comment:
This info including the `salespersonId` is internal. We should not include it in the bundle response.